operator/controllers/security.go

```diff
@@ -0,0 +1,47 @@
+package controllers
+
+import (
+	machinelearningv1 "github.com/seldonio/seldon-core/operator/apis/machinelearning.seldon.io/v1"
+	corev1 "k8s.io/api/core/v1"
+	"strconv"
+)
+
+func getSvcOrchUser(mlDep *machinelearningv1.SeldonDeployment) (*int64, error) {
+
+	if isExecutorEnabled(mlDep) {
+		if envExecutorUser != "" {
```

Comment:
If the env doesn't set a user, wondering if we should still have a default user?

Reply:
Openshift seems to add a default user automatically it was because we were explicitly setting a user that the openshift operator was failing as the user ID has different ranges in each namespace.